bundle/config/mutator/capture_schema_dependency_test.go

```diff
@@ -80,39 +38,16 @@ func TestCaptureSchemaDependencyForVolume(t *testing.T) {
 	d := bundle.Apply(context.Background(), b, CaptureSchemaDependency())
 	require.Nil(t, d)
 
-	assert.Equal(t, "${resources.schemas.schema1.name}", b.Config.Resources.Volumes["volume1"].CreateVolumeRequestContent.SchemaName)
-	assert.Equal(t, "${resources.schemas.schema2.name}", b.Config.Resources.Volumes["volume2"].CreateVolumeRequestContent.SchemaName)
-	assert.Equal(t, "${resources.schemas.schema3.name}", b.Config.Resources.Volumes["volume3"].CreateVolumeRequestContent.SchemaName)
-	assert.Equal(t, "foobar", b.Config.Resources.Volumes["volume4"].CreateVolumeRequestContent.SchemaName)
-	assert.Equal(t, "schemaX", b.Config.Resources.Volumes["volume5"].CreateVolumeRequestContent.SchemaName)
-
+	assert.Equal(t, "foobar", b.Config.Resources.Volumes["volume1"].CreateVolumeRequestContent.SchemaName)
 	assert.Nil(t, b.Config.Resources.Volumes["nilVolume"])
 	assert.Nil(t, b.Config.Resources.Volumes["emptyVolume"].CreateVolumeRequestContent)
 }
 
-func TestCaptureSchemaDependencyForPipelinesWithTarget(t *testing.T) {
+func TestCaptureSchemaDependencyForPipelinesWithEmptySchemas(t *testing.T) {
 	b := &bundle.Bundle{
 		Config: config.Root{
 			Resources: config.Resources{
 				Schemas: map[string]*resources.Schema{
-					"schema1": {
```

Comment:
I'm on the fence regarding whether we should remove this unit test here. On one hand, it serves as documentation regarding what the mutator does. On the other hand, it duplicates the acceptance test content.

More generally, what do you guys think about maintaining unit tests that are copies of acceptance tests that we have? cc: @pietern @andrewnester @denik

Reply:
I think those should be removed. If it's not adding coverage, it's just maintenance without the upside.